src/main/java/space/obminyashka/items_exchange/rest/LocationController.java

```diff
@@ -56,6 +56,13 @@ public List<LocationNameView> getAllAreas() {
         return locationService.findAllAreas();
     }
 
+    @GetMapping(value = ApiKey.LOCATION_DISTRICT, produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get all of existed districts by area id.")
+    @ResponseStatus(HttpStatus.OK)
+    public List<LocationNameView> getAllDistricts(@RequestParam("area_id") UUID areaId) {
```

Comment:
you can name variable areaId and remove () in @RequestParam

Reply:
++